docs/platform/release-notes/tizen-5-5-m3.md

```diff
@@ -0,0 +1,93 @@
+﻿# Tizen 5.5 Public M3 Release Notes
+
+Release date: Aug. 31, 2020
+
+
+- Tizen 5.5 public M3 adopts some new features for wearable, such as AOD viewer, Sticker framework, Aurum for testing-automation, and Battery-Monitor framework.

```

Comment:
The Tizen 5.5 Public release provides new features for wearable, such as AOD viewer, Sticker framework, Aurum for testing-automation, and Battery-Monitor framework.

@mijin85cho, should the lead line be standard?

Reply:
Release Details needs to be added.

Reply:
@mijin85cho  agree , I do not prefer this as a list, missed to mention it explicitly.

Reply:
will accept